tests/php/Dev/Tasks/NormaliseAccessMigrationHelperTest.php

```diff
@@ -74,6 +75,9 @@ protected function tearDown(): void
      */
     public function testSanityCheck()
     {
+        if (Deprecation::get_is_enabled()) {
```

Comment:
I possibly could have called `$this->markTestSkipped()` in `setUp()` - though I never tried. However even if this worked, I'd still prefer to call `$this->markTestSkipped()` on test individually so that we're being consistent with all the other unit-tests